decidim-initiatives/spec/controllers/decidim/initiatives/admin/initiatives_type_scopes_controller_spec.rb

```diff
@@ -10,7 +10,7 @@ module Admin
 
         let(:organization) { create(:organization) }
         let(:admin_user) { create(:user, :confirmed, :admin, organization:) }
-        let(:user) { create(:user, :confirmed, organization:) }
+        let(:user) { create(:user, :confirmed, admin_terms_accepted_at: Time.current, organization:) }
```

Comment:
You can also use the `:admin_terms_accepted` trait here.
```suggestion
        let(:user) { create(:user, :confirmed, :admin_terms_accepted, organization:) }
```